lib/ansible/modules/windows/win_iis_webbinding.py

```diff
@@ -28,74 +15,58 @@
 ---
 module: win_iis_webbinding
 version_added: "2.0"
-short_description: Configures a IIS Web site.
+short_description: Configures a IIS Web site binding.
 description:
      - Creates, Removes and configures a binding to an existing IIS Web site
 options:
   name:
     description:
       - Names of web site
     required: true
-    default: null
-    aliases: []
+    aliases: website
   state:
     description:
       - State of the binding
     choices:
       - present
       - absent
-    required: false
-    default: null
-    aliases: []
+    default: present
   port:
     description:
       - The port to bind to / use for the new site.
-    required: false
-    default: null
-    aliases: []
+    default: 80
   ip:
     description:
       - The IP address to bind to / use for the new site.
-    required: false
-    default: null
-    aliases: []
+    default: '*'
   host_header:
     description:
       - The host header to bind to / use for the new site.
-    required: false
-    default: null
-    aliases: []
+      - For state absent, you can use '*' here to remove all bindings for a particular
```

Comment:
`C(*)`
or `C('*')`  if it needs quoting

http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#formatting-options